### DISCUSSION TITLE
Chinese Version

### DISCUSSION THREAD

**Discussion by derekdylu** [monkeytypegame/monkeytype#2210] [Ideas]
from Taiwan! wish there could be a Traditional Chinese version!

**Comment by Miodec** [comment]
Do you mean typed language or ui language?

**Reply by derekdylu** [reply]
Type language

**Reply by Miodec** [reply]
I think there is chinese available already?

**Reply by LaoshuBaby** [reply]
> I think there is chinese available already?

![mandarin](https://user-images.githubusercontent.com/42690037/196477946-c23df073-7d9c-4292-8ce1-103f019ee85c.png)
![chinese](https://user-images.githubusercontent.com/42690037/196477998-86222186-6847-457e-97af-eac35f967038.png)

No, monkeytype currently haven't Chinese sample paragraph. (And I haven't found where can change UI language)

How can we add paragraph? Is there any introduction?

**Reply by ridemountainpig** [reply]
Hi @derekdylu @LaoshuBaby, The chinese is already avaiable, you change to chinese traditional mode like the video.

![CleanShot 2023-07-18 at 20 54 36@2x](https://github.com/monkeytypegame/monkeytype/assets/92412722/39646724-2b84-45e9-a190-5c591d1a27e0)

https://github.com/monkeytypegame/monkeytype/assets/92412722/41ffdc42-52b6-4278-aa01-a42c77837d6a
